webapi/webapi-speechapi-tizen-tests/speechapi/Speech_vocalizeString_setCBListener-manual.html

```diff
@@ -35,7 +35,46 @@
 <link rel="author" title="Intel" href="http://www.intel.com">
 <meta name="assert" content="Test checks that if vocalizeString and setCBListener work well.">
 <link rel="help" href="https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html">
+
+<h2>Preconditions</h2>
+<ol>
+  <li>
+    Plug in USB microphone
+  </li>
+  <li>Update "sphinx.pulsesrc" section of /etc/speech-recognition/speech-recognition.conf following the output of below command.<br/>
+      (app user): pactl list sources |grep input |grep "Name:" |grep usb
+  </li>
+  <li>
+    Reboot test device
+  </li>
+  <li>
+    Execute following command to init environment:<br/>
+    (app user) app_launcher -s org.tizen.dialer<br/>
+    (app user) pacmd set-sink-mute 0 false<br/>
+  </li>
+  <li>
+    Launch test app
+  </li>
+</ol>
+
+<p>TTS Test: Type in some word in the textbox,then press button "Speak Out".</p>
+<p>Test Passes if you can hear the word typed in speak out from headset.</p>
+<p>STT Test: Press "Record Voice", speak to mic some word then click "Stop Recording".</p>
```

Comment:
Hmm, prefer to make some simple words and let the tester read them after click the Record button. The results shall be the recorded words.